Classes/UVDeflection.m

```diff
@@ -90,14 +92,18 @@ + (NSInteger)interactionIdentifier {
 
 + (NSMutableDictionary *)deflectionParams {
     NSMutableDictionary *params = [NSMutableDictionary dictionary];
-    if ([UVBabayaga instance].uvts) {
+    UVBabayaga *inst = (UVBabayaga *)[UVBabayaga instance];
+    
+    if (inst && inst.uvts != nil) {
```

Comment:
The instance is a singleton, and cannot be null. There is no reason to check it here, nor is there any technical reason to put it into a variable.

The suggestion in #403 was to hold onto the actual string so that it can't get set to null in between the null check and where it is used.